src/meshes/p4est_mesh.jl

```diff
@@ -1021,9 +1136,267 @@ function cubed_sphere_mapping(xi, eta, zeta, inner_radius, thickness, direction)
 end
 
 
+# Calculate physical coordinates of each element of an unstructured mesh read
+# in from a HOHQMesh file. This calculation is done with the transfinite interpolation
+# routines found in `mappings_geometry_curved_2d.jl` or `mappings_geometry_straight_2d.jl`
+function calc_tree_node_coordinates!(node_coordinates::AbstractArray{<:Any, 4},
+                                     file_lines::Vector{String}, nodes, vertices, RealT)
+  # Get the number of trees and the number of interpolation nodes
+  n_trees = last(size(node_coordinates))
+  nnodes = length(nodes)
+
+  # Setup the starting file index to read in element indices and the additional
+  # curved boundary information provided by HOHQMesh.
+  file_idx = findfirst(occursin.("mesh polynomial degree", file_lines)) + 1
+
+  # Create a work set of Gamma curves to create the node coordinates
+  CurvedSurfaceT    = CurvedSurface{RealT}
+  surface_curves    = Array{CurvedSurfaceT}(undef, 4)
+
+  # Create other work arrays to perform the mesh construction
+  element_node_ids  = Array{Int}(undef, 4)
+  curved_check      = Vector{Int}(undef, 4)
+  cornerNodeVals    = Array{RealT}(undef, (4, 2))
+  tempNodes         = Array{RealT}(undef, (4, 2))
```

Comment:
```suggestion
  cornerNodeVals    = Array{RealT}(undef, (4, 2))
  tempNodes         = Array{RealT}(undef, (4, 2))
```
`snake_case` for consistency?

Reply:
:+1:

Reply:
Should I also update the variable names to snake case in the UnstructuredMesh2D constructor?

Reply:
Yes, please :+1:

Reply:
Done